cogs/torrents.py

```diff
@@ -0,0 +1,66 @@
+import os
+import discord
+import requests
+import json
+from discord.ext import commands
+
+
+def load_json(token):
+    with open('./config.json') as f:
+        config = json.load(f)
+    return config.get(token)
+
+
+def setup(client):
+    client.add_cog(Torrents(client))
+
+
+def get_magnet(search: str) -> discord.Embed:
+    """Get a magnet link for a torrent"""
```

Comment:
Add Usage to the help comments. See https://github.com/Joelpls/discord-bot/blob/1488dca517c77d15962b16358f473c8d53e70f83/cogs/memes.py#L37